transcriptic/config.py

```diff
@@ -722,6 +723,9 @@ def submit_launch_request(
             "test_mode": test_mode,
             "payment_method_id": payment_method_id,
         }
+        if predecessor_id:
```

Comment:
I think the `predecessor_id` can be added to the `payload` directly. There if a filter step after it, it will filter out the None value items anyway.

Reply:
agreed with Xi here, good catch!